packages/web/src/pages/upload-page/forms/EditCollectionForm.tsx

```diff
@@ -53,10 +55,12 @@ export const EditCollectionForm = (props: EditCollectionFormProps) => {
   const { isEnabled: isPremiumAlbumsEnabled } = useFeatureFlag(
     FeatureFlags.PREMIUM_ALBUMS_ENABLED
   )
+  const [downloadToggleValue, setDownloadToggleValue] = useState(false)
```

Comment:
Is there a reason not to track this in the form values? It would remove the need to pass the `onSubmit` down to `AccessAndSaleField`